src/NServiceBus.Core/Outbox/OutboxSendBehavior.cs

```diff
@@ -15,7 +15,15 @@ public void Invoke(OutgoingContext context, Action next)
 
             if (context.TryGet(out currentOutboxMessage))
             {
-                currentOutboxMessage.TransportOperations.Add( new TransportOperation(context.OutgoingMessage.Id, context.DeliveryOptions.ToTransportOperationOptions(), context.OutgoingMessage.Body, context.OutgoingMessage.Headers)); 
+                var options = context.DeliveryOptions.ToTransportOperationOptions();
+                var transportOperation = new TransportOperation(context.OutgoingMessage.Id, options, context.OutgoingMessage.Body, context.OutgoingMessage.Headers);
+
+                if (context.OutgoingMessage.TimeToBeReceived != TimeSpan.MaxValue)
+                {
+                    options["TimeToBeReceived"] = context.OutgoingMessage.TimeToBeReceived.ToString();
```

Comment:
Yea, think we're ok. We just a plain TimeSpan.Parse here as well https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Outbox/TransportOperationConverter.cs#L39

This highlights another issue: There is 2 ways to change TTBR in v5, via the sendoptions or straight on the TransportMessage. V6 has cleaned all this up so I propose we leave this as is